CMakeLists.txt

```diff
@@ -130,6 +130,12 @@ if(TOKEN_AUTH_ONLY)
    add_definitions(-DTOKEN_AUTH_ONLY=1)
 endif()
 
+option(WITH_WEBENGINE "Enable old webflow v1 auth using webengine" ON)
```

Comment:
Please rename, what you mean is Web Flow v1 auth enabled ?
This has nothing to do with web engine.